docs/release_notes/release_notes.md

```diff
@@ -8,61 +8,73 @@ has_children: true
 
-    * Network Reports   
-        * Added drilldown to the `Port Scanning Attempts` map.
+* ### New Vulnerability Dashboard
 
-    * Office 365
-        * Updated the alerts and dashboard to use the new `o365:service:healthIssue` sourcetype (`o365:service:status` sourcetype has been retired by the Add-on).
-        * Added `Login by location` map to the `Office 365` dashboard.
-        * Added `authentication_method` and `user_type` fields for O365 login related alerts and dashboard panels.
+    * Replaced `Tenable` and `Qualys` dashboard with a new `Vulnerability` dashboard.
+
+* ### Added New Alerts
+
+   * Authentication
+      * `Authentication - Long Running VPN Session Disconnected`
+
+* ### Enhancements 
 
-    * Ransomware Alerts
-        * Enhanced filters for paths to reduce false positives for both `Ransomware - Calculate UpperBound for Spike in File Writes` and `Ransomware - Spike in File Writes`.
+    * Network Reports
+        * Updated Map chart from `network_telemetry_map` to Splunk Map to show all the traffic instead of top 20 traffic detail.
 
+    * Palo Alto Firewall
+        * Added `dvc_name` field in the `List of Firewall Devices` dashboard panel.
+    
     * VPN 
-        * Added `Successful Session` dashboard panel.
-        * Added drilldown to the `Connected Workforce by Location` map.
+        * Added `Elapsed Time Per Session` dashboard panel.
+
+    * Updated splunklib to the latest version (v1.7.0)
 
-    * Windows Reports 
-        * Added `Listening Ports on Host` dashboard panel to the `Windows Reports` dashboard. For data collection, users need to enable the `win_listening_ports` scripted input.
+    * Renamed below eventtypes to follow consistence naming convention.
+        | Before | Now |
+        |---|---|
+        | cs_palo_gp_old_login | cs_palo_gp_old_vpn_login |
+        | cs_palo_gp_old_logout | cs_palo_gp_old_vpn_logout |
+        | cs_palo_gp_new_login | cs_palo_gp_new_vpn_login |
+        | cs_palo_gp_new_connected | cs_palo_gp_new_vpn_connected |
+        | cs_palo_gp_new_logout | cs_palo_gp_new_logout |
 
+    * Added `src_ip`, `src` and `private_ip` field extraction for the Cisco VPN logoff event.
+
+    * Added `cs_fortigate` macro.
 
 * ### Bug Fixes
-    * Fixed a drilldown issue for the `Login Details` dashboard panel in the `VPN` dashboard.
 
-    * Fixed the source value in the `cs_sysmon macro` macro.
+    * Forensics
+        * Resolved the Search dropdown issue for `O365 - Azure Active Directory -*` dropdown values.
+
+    * Office 365
+        * Resolved the duplicate event issue for O365 management activity related alerts and dashboard.
+
+
 
+## Upgrade Guide from 2.2.0 to 2.3.0
 
-## Upgrade Guide from 2.1.0 to 2.2.0
+* After App Upgrade, Run the `Asset Inventory - Vulnerability Lookup Gen` and `Asset Inventory - Lookup Gen` with last 1 year time range to populate `cs_all_assets` and `cs_all_vuln` lookups with historical data.
```

Comment:
After App Upgrade, Run the `Asset Inventory - Vulnerability Lookup Gen` and `Asset Inventory - Lookup Gen` with the last 1-year or longer time range as necessary to populate the historical data in the `cs_all_assets` and `cs_all_vuln` lookups. Make sure to use `summariesonly=false` in the search to cover all the data.